crypto/CMakeLists.txt

```diff
@@ -110,12 +110,20 @@ else()
   elseif(APPLE)
     set(ASSEMBLY_SOURCE ${GENERATE_CODE_ROOT}/mac-${ARCH}/crypto/)
   elseif(UNIX)
-    set(ASSEMBLY_SOURCE ${GENERATE_CODE_ROOT}/linux-${ARCH}/crypto/)
+    if(${ARCH} STREQUAL "generic")
+      message(STATUS "Detected generic linux platform. No assembly files will be included.")
+    else()
+      set(ASSEMBLY_SOURCE ${GENERATE_CODE_ROOT}/linux-${ARCH}/crypto/)
+    endif()
   elseif(WIN32)
     set(ASSEMBLY_SOURCE ${GENERATE_CODE_ROOT}/win-${ARCH}/crypto/)
   endif()
-  message("Copying platform assembly files from ${ASSEMBLY_SOURCE} to ${CMAKE_CURRENT_BINARY_DIR}")
-  file(COPY ${ASSEMBLY_SOURCE}/ DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+
+  # Only attempt to copy pre-generated files from
```

Comment:
from... (something missing?)

Reply:
Forgot to clean this up. Should be fixed with most recent commit 😄